mm2src/coins/z_coin.rs

```diff
@@ -417,6 +423,7 @@ impl SwapOps for ZCoin {
             other_pub,
             secret_hash,
             tx,
+            0,
```

Comment:
Will it be a different vout index from the default one?

Reply:
No, I forgot to update it, just one second :slightly_smiling_face: